content/post/057-ipfs-camp-course-videos.md

```diff
@@ -0,0 +1,80 @@
+
+[![Video: The Lifecycle of Data in DWeb](/img/057-ipfs-camp-course-videos/core-d-thumbnail.png)](https://youtu.be/fLUq0RkiTBA)
+
+_Video: [The Lifecycle of Data in DWeb](https://youtu.be/fLUq0RkiTBA)_
+
+
+
+### And more!
+
+There are plenty more core and elective courses to explore in the latest batch of videos. Check out our full [course playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtSsHMhjeWpfOzr8tonPaePu) or jump right to the video that looks most interesting:
+
+**Core Courses**
+
+- [Understanding how the InterPlanetary File System deals with files](https://youtu.be/dJ7TkB4UpZ8)
+- [Solving distributed networking problems with libp2p](https://youtu.be/Q4IH5rWEO-E)
+- [Developing apps with IPFS API](https://youtu.be/6EHBOnFQJN4)
+- [The lifecycle of data in DWeb](https://youtu.be/fLUq0RkiTBA)
+
+**Elective Courses**
+
+- [Identity on the DWeb using IDM and Nomios](https://youtu.be/EsLqjFEN-MM)
+- [Managing pinsets with IPFS Cluster](https://youtu.be/wmxaVrAFfeE)
+- [Cat Roulette! Building a p2p laser catbot](https://youtu.be/kVzHE4MRPOM)
+- [Deploying IPFS Infrastructure](https://youtu.be/PD0e89b4NBk)
+- [Starting, Growing, and Stewarding your IPFS Community](https://youtu.be/crTa1j3FRac)
+- [Building DApps with Textile, the iCloud for the DWeb](https://youtu.be/4r7_lOyv4W8)
+- [Managing Datasets with QRI](https://youtu.be/ZN1mPBECsvA)
+
+
+Each of these courses has its own folder in the IPFS Camp repo, complete with course descriptions, slide decks, videos, code walk-throughs, or other materials provided by our awesome instructors.  Check out the [complete course listings](https://github.com/ipfs/camp#-core--elective-courses) for all the details.
+
+## Explore related content in ProtoSchool
+If you enjoy hands-on learning, set down that popcorn for a minute and try out a [ProtoSchool tutorial](https://proto.school/#/tutorials)! Or better yet, find a [chapter near you](https://proto.school/#/chapters) and attend a live workshop. (Want to get really meta? Today's batch of videos includes an [interactive workshop on leading interactive workshops](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_F)!)
+
+The ProtoSchool curriculum is growing, and we're excited to explore all of the fabulous IPFS Camp content to find gems to add to our tutorials. Watch [this issue](https://github.com/ProtoSchool/protoschool.github.io/issues/261) to track those content developments, or learn more about [contributing to ProtoSchool](https://proto.school/#/contribute) as a tutorial author or chapter leader.
+
+## But wait, that's not all!
+There's even more great content from IPFS Camp content coming your way, including recordings of keynotes and interviews. If you're interested in receiving an update when the next batch of videos are available, try out one of these strategies:
```

Comment:
Big fan of this plug! Do we do this everywhere? Because we should...

Reply:
If you mean the references to how to subscribe to updates, then yes, I stole this from other recent camp-related posts. :)